STM32F405RGTx/Core/Src/rtos_threads/svpwm_thread.cpp

```diff
@@ -0,0 +1,100 @@
+#include <stdio.h>
+#include <stdint.h>
+#include "main.h"
+#include "threads.hpp"
+#include "math.h"

```

Comment:
Use angle brackets instead of quotations marks for standard library includes, e.g.,

```C
#include <math.h>
```

Using angle brackets will search the designated directory in your OS for standard library C header files first, while quotation marks will search the include directories in the makefile first.

The angle brackets is best practice for including standard library headers.

Reply:
changed